packages/communication-react/package.json

```diff
@@ -62,12 +62,14 @@
   "main": "./dist/dist-cjs/communication-react/index.js",
   "module": "./dist/dist-esm/communication-react/src/index.js",
   "scripts": {
+    "prepare": "ts-patch install -s",
```

Comment:
What does this new prepare command do?

Reply:
Actually you're right I think this is not needed, we do live transformations during build (there is a fancy option to do transformations at install time but not what we're looking for). I'll try remove and see if it works

Reply:
removed